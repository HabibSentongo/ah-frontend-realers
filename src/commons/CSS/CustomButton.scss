@import '../assets/variables/colors.scss';
@import '../assets/variables/font.scss';

.custom-btn {
  background-color: $primary-blue !important;
  border-color: lighten($primary-blue, 10%) !important;
  border-radius: 0%;
  color: $white;
  font-family: $source-san-font;
  font-size: 18px;
  font-weight: 100;
  height: 41px;
}
