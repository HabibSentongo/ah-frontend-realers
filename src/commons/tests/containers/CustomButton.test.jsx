/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import CustomButton from '../../containers/CustomButton';

// test
describe('CustomButton Component', () => {
  it('should match the snapshot', () => {
    const component = shallow(<CustomButton />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
