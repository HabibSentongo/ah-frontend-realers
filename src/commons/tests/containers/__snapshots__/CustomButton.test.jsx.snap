// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton Component should match the snapshot 1`] = `
<Fragment>
  <Button
    active={true}
    className="custom-btn"
    disabled={false}
    href="/articles"
    type="button"
    variant="primary"
  >
    stranger
  </Button>
</Fragment>
`;
